packages/dependency-profiles/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@fluentui-react-native/dependency-profiles",
+  "version": "0.0.0",
+  "description": "@rnx-kit/dep-check profiles covering packages published from FluentUI-React-Native",
+  "license": "UNLICENSED",
```

Comment:
We should have a license defined. I think most packages use MIT?

Reply:
Fixed. Thanks!